### ISSUE TITLE
task placement constraints

### ISSUE THREAD

**Issue by tobowers** [aws/amazon-ecs-cli#212]
http://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-placement-constraints.html is a pretty key thing for real-world services. 

I could take a crack at supporting them, but was wondering if there are any design documents already in place. 

Other orchestrators tend to use labels from the docker compose in order to facilitate this kind of thing... 

something like

```yaml
labels:
  - com.amazon.ecs.constraints.type=memberOf
  - com.amazon.ecs.constraints.expression="not(task:group == database)"
```

Others use something separate like a "rancher-compose.yml" which gets merged into the main document and supports custom attributes.

**Comment by yinshiua** [comment]
We do not have a design for task placement in place, and planed in our next couple releases. Feel free to submit a design proposal on how the user experience would be to so we can make sure this is going in the right direction. Thanks!

**Comment by SoManyHs** [comment]
Hi, we have this on our roadmap and will keep you posted. Thanks!

**Comment by SoManyHs** [comment]
This is now available in 1.7.0. Closing.
